bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -314,7 +340,7 @@
 	<channel-type id="vehicleName">
 		<item-type>String</item-type>
 		<label>Vehicle Name</label>
-		<description>Name of EV (former vehicle-id)</description>
+		<description>Name of EV</description>
```

Comment:
I’m not sure whether this description could be improved to make clear it is used to identify the vehicle. Compared with openHAB, vehicle name is like Item name and vehicle title is like Item label.

Probably:

```suggestion
		<description>Name of EV (unique identifier of the EV containing no whitespaces or special characters)</description>
```

Reply:
> The unique identifier of the EV used in the evcc configuration (containing no whitespaces nor special characters)